src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -214,21 +214,21 @@
     (mq/stop-broker! broker)))
 
 (defn stop-puppetdb
-  "Callback to execute any necessary cleanup during a normal shutdown."
-  [context]
-  (log/info "Shutdown request received; puppetdb exiting.")
-  (shutdown-mq-broker context)
-  context)
-
-(defn error-shutdown!
-  "Last-resort shutdown/cleanup code to execute when a fatal error has occurred."
-  [context]
-  (log/error "A fatal error occurred; shutting down all subsystems.")
+  "Shuts down PuppetDB, releasing resources when possible; emergency?
+  should not be set for a normal shutdown."
```

Comment:
The semicolon should be a period. Also what does emergency? do? Might as well explain if we're mentioning it.